marketplace/typeform/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "typeform",
+  "version": "1.0.0",
+  "private": true,
+  "devDependencies": {
+    "@babel/core": "7.4.5",
+    "@babel/plugin-proposal-class-properties": "7.4.4",
+    "@babel/plugin-transform-runtime": "7.4.4",
+    "@babel/preset-env": "7.4.5",
+    "@babel/preset-react": "7.0.0",
+    "@contentful/contentful-extension-scripts": "0.11.0",
+    "contentful-cli": "^0.29.0",
+    "jest": "^24.8.0",
+    "react-test-renderer": "^16.8.6"
+  },
+  "dependencies": {
+    "@contentful/forma-36-fcss": "^0.0.20",
+    "@contentful/forma-36-react-components": "^3.2.5",
+    "@contentful/forma-36-tokens": "^0.3.0",
+    "contentful-ui-extensions-sdk": "^3.5.0",
+    "corejs": "^1.0.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "whatwg-fetch": "^3.0.0"
+  },
+  "scripts": {
+    "configure": "contentful space use && contentful space environment use",
+    "start": "contentful-extension-scripts start",
+    "build": "contentful-extension-scripts build",
+    "deploy": "npm run build && contentful extension update --force",
+    "login": "contentful login",
+    "logout": "contentful logout",
+    "help": "contentful-extension-scripts help",
+    "test": "TZ=UTZ jest test/*.test.js",
```

Comment:
Should this be `TZ=UTC`? 

(same comment for the line below)

Reply:
Good catch. Fixed!